### PR TITLE
Remove unnecessary layout animation of `Switch` component

### LINKED ISSUE #3168: Thumb of `Switch` on `Menu` animation
### Which package has an issue?

@yamada-ui/react

### Package Version

1.5.4

### Description

The thumb of `Switch` on `Menu` is strange.
This happens when you first open `Menu', because the initial position of `MenuList' is at the top left of the screen.
This is due to the fact that the thumb animation starts from the initial position even if it was hidden.

https://github.com/user-attachments/assets/a81f8218-e59b-4755-a7be-5798fa6d285e

### Link to Reproduction

_No response_

### Steps to Reproduce

1. Set `Switch` on `Menu`
2. Open `Menu`

### Browser

Chrome

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by 108yen**
Closes #3168

## Description

Removed unnecessary layout animation so that when the `Switch` component itself transitions, the thumb animation renders correctly.

## Is this a breaking change (Yes/No):

No
## Additional Information
https://www.framer.com/motion/layout-animations/#troubleshooting

### CODE REVIEW

**Review by neginasu — APPROVED** [review]

File: packages/components/switch/src/switch.tsx [hunk 1]
```diff
@@ -123,26 +123,28 @@ export const Switch = forwardRef<SwitchProps, "input">((props, ref) => {
...
-        >
-          <motion.span
-            className={cx("ui-switch__thumb", className)}
-            data-checked={dataAttr(isChecked)}
-            layout
-            transition={transition}
-            __css={{ ...styles.thumb }}
-          />
-        </ui.span>
+        <motion.div layout layoutRoot>
```
Comment (hirotomoyamada) [inline packages/components/switch/src/switch.tsx:126]: @108yen 

Why does `motion.div` `layout` and `layoutRoot` solve this problem?

Comment (108yen) [reply packages/components/switch/src/switch.tsx:126]: @hirotomoyamada 
Check this.
https://www.framer.com/motion/layout-animations/#troubleshooting

Comment (hirotomoyamada) [reply packages/components/switch/src/switch.tsx:126]: Why don't you set `layout` and `layoutRoot` on the `span` element?

Comment (108yen) [reply packages/components/switch/src/switch.tsx:126]: @hirotomoyamada 

The `getIconProps()` result type is different from the `motion.span` prop type.

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**